about.html

```diff
@@ -43,78 +43,67 @@ <h2>Our Story</h2>
         </p>
+        {% for sociallink in currBio.sociallinks %}
+           {% if forloop.first %}<div class="social_links">{% endif %}
+           <a target="_blank" href="{{sociallink.link}}" aria-label="{{currBio.name}} {{sociallink.label}}">
+             <i class="{{sociallink.inconclass}}"></i>
+           </a>
+           {% if forloop.last %}</div>{% endif %}
+        {% endfor %}
       </div>
 
-      <div class="profile-left">
-        <img alt="KC Jones Evans" src="{{site.baseurl}}/img/people/kcjonesevans.jpg" />
-        <h3>K.C. Jones-Evans, Chapter Co-Lead</h3>
-        <div class="profile-links">
-          <a target="_blank" href="https://twitter.com/kcjonesevans" aria-label="K.C. Jones-Evans Twitter"><i class="profile-icon fa fa-twitter"></i></a>
-          <a target="_blank" href="https://www.linkedin.com/in/kcjonesevans/" aria-label="K.C. Jones-Evans LinkedIn"><i class="profile-icon fa fa-linkedin-square"></i></a>
-          <a target="_blank" href="http://kcjonesevans.com" aria-label="K.C. Jones-Evans Website"><i class="profile-icon fa fa-globe"></i>
-          <a target="_blank" href="mailto:kc@girldevelopit.com" aria-label="K.C. Jones-Evans Email"><i class="profile-icon fa fa-envelope"></i>
-        </div>
-        <p>
-          K.C. Jones-Evans is a software developer with Applied Information Sciences. She has a B.S. in Computer Science and over 6 years of experience in the complete development cycle of software, from initial requirement gathering to maintenance of completed systems. K.C. enjoys spending her free time doing CrossFit, crocheting, discovering new food spots, and of course playing on her computer.
-        </p>
-      </div>
+      {% endfor %}
+
     </div>
 
+{% if site.data.about_bios.staff %}
     <div class="container">
       <div class="headline">
         <img alt="Flourish" src="{{ site.baseurl }}/img/flourish.png" />
         <h2>Our Staff</h2>
       </div>
 
-      <div class="profile-left">
-        <img alt="Catherine Meade" src="{{site.baseurl}}/img/people/catmeade.jpg" />
-        <h3>Catherine Meade, Instructor</h3>
-        <div class="profile-links">
-          <a target="_blank" href="https://twitter.com/catheraaine" aria-label="Catherine Meade Twitter"><i class="profile-icon fa fa-twitter"></i></a>
-          <a target="_blank" href="https://www.linkedin.com/in/catheraaine/" aria-label="Catherine Meade LinkedIn"><i class="profile-icon fa fa-linkedin-square"></i></a>
-          <a target="_blank" href="http://catheraaine.com" aria-label="Catherine Meade Website"><i class="profile-icon fa fa-globe"></i>
-        </div>
-        <p>
-          Catherine is a web developer at Sparkbox, and teaches Intro to HTML/CSS with GDI. She has a B.S. in Art Education and a strong passion for teaching. Cat's other favorite things include reading, video games, tabletop games, oil painting, and boffer sports.
-        </p>
+      {% for currBio in site.data.about_bios.staff %}
```

Comment:
Somehow Kevin's bio is being labeled "profile-left", which is why I think we're seeing this:
![image](https://user-images.githubusercontent.com/12739849/37235998-950ca462-23d1-11e8-9f06-f193089a71fa.png)